.travis.yml

```diff
@@ -3,16 +3,16 @@ sudo: false
 cache:
   directories:
     - $HOME/.cache/pip
-python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
-  - "3.7-dev"
-  - "pypy-5.6.0"
 matrix:
-  allow_failures:
-    - python: "3.7-dev"
+  include:
+    - python: '2.7'
+    - python: '3.4'
+    - python: '3.5'
+    - python: '3.6'
+    - python: '3.7'
+      dist: xenial
+      sudo: true
```

Comment:
So instead of doing this, I suggest enabling sudo+xenial globally because Travis CI is shutting down their container-based infra anyway.